pkg/authentication/options/options.go

```diff
@@ -30,6 +30,7 @@ import (
 )
 
 type AuthenticationOptions struct {
+	AuthenticateRateLimiterEnable   bool           `json:"authenticateRateLimiterEnable" yaml:"authenticateRateLimiterEnable"`
```

Comment:
`AuthenticateRateLimiterEnable`  should removed. i don't think this field is needed.